images-list

```diff
@@ -1,6 +1,20 @@
-# This list should be sorted alphabetically. Do not change existing entries. Comments lines may be started with # or //.
+# This list should be sorted lexographically. Do not change existing entries. Comments lines may be started with # or //.
```

Comment:
Thanks for syncing this up with the PR template. I think I was waffling as to whether or not people would know what this meant ;)

Reply:
I did have to google because I couldn't remember all the rules 😄

Reply:
I think since we're including non-alphabetical characters (such as whitespace, punctuation, and digits) in the sort it's not technically in alphabetical order... so referring to it as lexographic order is technically correct - the best kind of correct!

Reply:
I also had to google it 😆 . Since it's not a big deal if we accidentally fall into alphabetical instead of lexographical, let's  include alphabetical somewhere in here. Like: 
```suggestion
# This list should be sorted lexographically (alphabetically). Do not change existing entries. Comments lines may be started with # or //.
```

Reply:
Alright, I went with caleb's approach so that you get a general idea of what to do, but could google for more accurate sorting.